aqi_monitor/apps/monitor/function_for_task.py

```diff
@@ -0,0 +1,33 @@
+from datetime import datetime as DT, timedelta
+
+def out_message(aqi):
+    if aqi < 51:
+        return "Good air"
+    elif aqi < 101:
+        return "Air quality is acceptable"
+    elif aqi < 151:
+        return "Members of sensitive groups may experience health effects"
+    elif aqi < 201:
+        return "Everyone may begin to experience health effects"
+    elif aqi < 301:
+        return "Health warnings of emergency conditions"
+    else:
+        return "Health alert: everyone may experience more serious health effects"
+
+def time_convert(time):
+    if time["tz"][0] == "+":
+        hour, minute = int(time["tz"][1:3]), int(time["tz"][4:])
+    else:
+        hour, minute = int(time["tz"][0:3]), int("-" + time["tz"][4:])
+    date = DT.strptime(time["s"], "%Y-%m-%d %H:%M:%S") + \
+           timedelta(hours=hour, minutes=minute)
+
+    return date.strftime("%Y-%m-%d %H:%M:%S")
```

Comment:
1) Please raname function to 'to_utc'

2) Add type annotation

3) Write docstring

4) Implementation is too complicated.   We can avoid this and concatenate time with tz, convert it to datetime object and perform convertion to uts format.

Reply:
Fixed